sirepo/util.py

```diff
@@ -235,6 +253,10 @@ def random_base62(length=32):
     return ''.join(r.choice(numconv.BASE62) for x in range(length))
 
 
+def safe_join(*paths):
```

Comment:
Let's call this `safe_path`, because it's unclear what's being joined in safe_join, and we can also see quickly when a safe_join needs to be updated to safe_path.